src/main/kotlin/com/terraformation/backend/accelerator/variables/StableId.kt

```diff
@@ -0,0 +1,50 @@
+package com.terraformation.backend.accelerator.variables
+
+import com.terraformation.backend.db.default_schema.LandUseModelType
+
+enum class StableId(val value: String) {
```

Comment:
I'm a little torn on making this an enum as opposed to an `object` with a bunch of `val` properties. The enum gives us type safety, but it means that the `StableId` isn't the representation of a "stable ID" in the general sense, just of the specific ones we care about. What would this look like if `StableId` were a wrapper around an arbitrary string value, the way our other ID classes like `UserId` wrap an underlying value? And then these constants could either be declared in its companion object or in a separate `StableIds` object, e.g.,
```kotlin
object StableIds {
  val Country = StableId("1")
}
```
With this approach, we could use the `StableId` type everywhere we need to represent any stable ID, whether or not it's one of the ones we care about in applications.